### PR TITLE
Create ui-c8y-1020-0-0-extract-the-reports-view-into-a-separate-plugin-5966.md

### PR DESCRIPTION
**Pull request by gen-App-Enablement**
Release note created by jdre

Ticket: [MTM-58448](https://cumulocity.atlassian.net/browse/MTM-58448)
Original PR: [5966](https://github.softwareag.com/IOTA/cumulocity-ui/pull/5966)

### CODE REVIEW

File: content/change-logs/application-enablement/ui-c8y-1020-0-0-extract-the-reports-view-into-a-separate-plugin-5966.md [hunk 1]
```diff
@@ -0,0 +1,17 @@
+---
+date: ""
+title: Extract the Reports view into a separate plugin (#5966)
```
Comment (gen-App-Enablement) [inline content/change-logs/application-enablement/ui-c8y-1020-0-0-extract-the-reports-view-into-a-separate-plugin-5966.md:3]: The Claude AI suggested to use the following title:
```suggestion
title: Reports view extracted into separate plugin
```

File: content/change-logs/application-enablement/ui-c8y-1020-0-0-extract-the-reports-view-into-a-separate-plugin-5966.md [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+component:
+  - value: component-YbYJ3gLU_
+    label: Web SDK
+build_artifact:
+  - value: tc-pjJiURv9Y
+    label: ui-c8y
+ticket: MTM-58448
+version: 1020.0.0
+---
+Extract the Reports view into a separate plugin (#5966)
```
Comment (gen-App-Enablement) [inline content/change-logs/application-enablement/ui-c8y-1020-0-0-extract-the-reports-view-into-a-separate-plugin-5966.md:17]: The Claude AI suggested to use the following body:
```suggestion
To improve the modularity and maintainability of the Cumulocity IoT platform, the Reports view has been extracted into a separate plugin. This change allows for better separation of concerns and enables the Reports functionality to be developed and maintained independently from the core platform. Existing tenants and users should not experience any disruption, as the Reports view will continue to function as before. However, the plugin will need to be installed and enabled separately if it is not already available.
```

**Review by gen-App-Enablement — COMMENTED** [review]
AI improvements

File: content/change-logs/application-enablement/ui-c8y-1020-0-0-extract-the-reports-view-into-a-separate-plugin-5966.md (hunk 1 above)
Comment (jdreSAG) [inline content/change-logs/application-enablement/ui-c8y-1020-0-0-extract-the-reports-view-into-a-separate-plugin-5966.md:3]:
```suggestion
title: Reports page extracted into a separate plugin
```

File: content/change-logs/application-enablement/ui-c8y-1020-0-0-extract-the-reports-view-into-a-separate-plugin-5966.md (hunk 2 above)
Comment (jdreSAG) [inline content/change-logs/application-enablement/ui-c8y-1020-0-0-extract-the-reports-view-into-a-separate-plugin-5966.md:17]:
```suggestion
Report module is extracted from the Cockpit application and added as a separate plugin.
This is a first step towards removing the reports module entirely and replacing it with the dashboard manager.
This change might be a breaking change as the reports module is no longer a part of `@c8y/ngx-components/context-dashboard`, but is be part of `@c8y/ngx-components/report-dashboard`.

This change does not affect the layout and UX of the Cockpit application nor does it add any new functionalities.
As a side effect, a performance improvement might be noticed as reports from now on will be loaded lazily (that is, only when users navigate to the **Reports** page).
```

**Comment by jdreSAG** [comment]
I've added changes according to this change announcement, see https://github.com/SoftwareAG/c8y-docs/pull/1995

File: content/change-logs/application-enablement/ui-c8y-1020-0-0-extract-the-reports-view-into-a-separate-plugin-5966.md [hunk 3]
```diff
@@ -0,0 +1,22 @@
...
+component:
+  - value: component-YbYJ3gLU_
+    label: Web SDK
+build_artifact:
+  - value: tc-pjJiURv9Y
+    label: ui-c8y
+ticket: MTM-58448
+version: 1020.0.0
+---
+Report module is extracted from the Cockpit application and added as a separate plugin.
```
Comment (BeateRixen) [inline content/change-logs/application-enablement/ui-c8y-1020-0-0-extract-the-reports-view-into-a-separate-plugin-5966.md:17]:
```suggestion
The Report module has been extracted from the Cockpit application and added as a separate plugin.
```

File: content/change-logs/application-enablement/ui-c8y-1020-0-0-extract-the-reports-view-into-a-separate-plugin-5966.md [hunk 4]
```diff
@@ -0,0 +1,22 @@
...
+component:
+  - value: component-YbYJ3gLU_
+    label: Web SDK
+build_artifact:
+  - value: tc-pjJiURv9Y
+    label: ui-c8y
+ticket: MTM-58448
+version: 1020.0.0
+---
+The Report module has been extracted from the Cockpit application and added as a separate plugin.
```
Comment (BeateRixen) [inline content/change-logs/application-enablement/ui-c8y-1020-0-0-extract-the-reports-view-into-a-separate-plugin-5966.md:17]:
```suggestion
As announced earlier, the report module has been extracted from the Cockpit application and added as a separate plugin.
```

**Review by BeateRixen — APPROVED** [review]

**Review by jdreSAG — APPROVED** [review]

**Pull request merged**